README.md

```diff
@@ -9,4 +136,245 @@ The website for the Civic Tech Index
+```
+
+-Use the `git status` command to see what files are staged. 
+
+This command will list the files that have been staged.  These are the files that will be committed (saved) when you run the next command, `git commit`. 
+```bash
+git status
+```
+
+-Use the `git commit` command
+
+This command saves your work, and prepares it to push to your repository.  Use the `-m` flag to quickly add a message to your commit. Your message should be a short description of the issue you are working.  It will be extremely helpful if other people can understand your message, so try to reisst the temptation to be overly cryptic.
+
+To commit your changes with a message, run:
+```bash
+git commit -m “insert message here”
+```
+
+Congratulations!  You are now ready to push your work to your repository. 
+
+#### Step 7 Check upstream before you push
+
+Before you push your local commits to your repository, check to see if there have been updates made in the main Hack For LA website repository. `git fetch` will check remote repositories for changes without altering your local repository.
+
+```bash
+git fetch upstream
+```
+
+##### Step 7a No changes in the upstream repository
+
+If you do not see any output, there have not been any changes in the
+main CTI Front End website repository since the last time you
+checked. So it is safe to push your local commits to your fork.
+
+If you just type `git push` you will be prompted to create a new branch in your GitHub repository. The more complete command below will create a new branch on your copy of the website repository, and then push your local branch there. The name at the end of this command should be the same as the name of the local branch that you created back in step 6, as in the example below:  
+
+```bash
+git push --set-upstream origin fix-logo-width-311
+```
+
+##### Step 7b conflicting changes in the upstream repository
+
+When you check the upstream repository, you may see output like this:
+
+```bash
+Fetching upstream
+remote: Enumerating objects: 11, done.
+remote: Counting objects: 100% (11/11), done.
+remote: Compressing objects: 100% (7/7), done.
+remote: Total 11 (delta 5), reused 7 (delta 4), pack-reused 0
+Unpacking objects: 100% (11/11), 8.25 KiB | 402.00 KiB/s, done.
+From https://github.com/CTI-website-frontend/website
+ + 770d667...14f9f46 Bonnie     -> CTI-website-frontend/Bonnie  (forced update)
+ * [new branch]      bonnie     -> CTI-website-frontend/bonnie
+   5773ebe..0c86ecd  gh-pages   -> CTI-website-frontend/gh-pages
+```
+
+You can safely ignore changes in other issue branches, such as
+`bonnie` above. But if you see changes in gh-pages, as in
+`5773ebe..0c86ecd  gh-pages   -> CTI-website-frontend/gh-pages`, you should
+incorporate those changes into your repository before merging or
+rebasing your issue branch. Use the [instructions below](#incorporating-changes-from-upstream)
+to bring your fork up to date with the main repository.
+
+
+### Incorporating changes from upstream
```

Comment:
I haven't gotten to a point yet on the first ticket I'm working to try this section of instructions, but the process looks well documented and clear. Excited to try it out.